src/test/java/CreateTableTest.java

```diff
@@ -379,6 +379,28 @@ public void existingStageCreateTableGenerateCommandTest() throws Exception
                      "location=@aStage/to/table file_format=(TYPE=CSV) AUTO_REFRESH=false COMMENT='Generated with Hive metastore connector (version=null).';",
                  commands.get(0));
     assertEquals("Unexpected number of commands generated", 1, commands.size());
+
+    // Test with a hive db name not in the configured schema list
+    table.setDbName("someOtherDB");
+    createTableEvent =
+       new CreateTableEvent(table, true, TestUtil.initializeMockHMSHandler());
+
+    createExternalTable =
+       new CreateExternalTable(createTableEvent, mockConfig);
+
+    // Mock Snowflake client to return a location for this stage
+    TestUtil.mockSnowflakeStageWithLocation("s3://bucketname/path", "someSchema1");
+
+    commands = createExternalTable.generateSqlQueries();
+    assertEquals("generated create stage command does not match " +
```

Comment:
nit. "generated create external table command does not match expected create external table command"

Reply:
Oh, I'll fix the above one too.